app/models/i14y_document.rb

```diff
@@ -27,9 +30,10 @@ def self.create(attributes)
 
   def save
     run_callbacks :save do
-      params = attributes.reject{ |_k,v| v.blank? }
+      params = attributes.reject { |_k, v| v.blank? }
```

Comment:
Do you know if v is just nil? if so you can use .compact.

Reply:
I don't see a Rubocop warning re. parentheses for that line (and there isn't anywhere for them to go)...? (Note - I fixed a few things, but there were a few other Rubocop warnings for that file that I decided to defer to another round of refactoring so that we could get this out the door. ) 

`v` could be `nil` or an empty string, so `.compact` won't work.

Reply:
Click on view file to see the ruboco errors.  
<img width="1089" alt="screen shot 2019-02-26 at 1 58 58 pm" src="https://user-images.githubusercontent.com/7727136/53438847-c43eb180-39ce-11e9-86bd-3d564f5ce484.png">

Reply:
Discussed this and the error was for the line below which was not a code change for this review.